### ISSUE TITLE
[Question] Whether there are plans to support QWEN2-VL format？

### ISSUE THREAD

**Issue by wangyt42** [mlc-ai/mlc-llm#2878]
## ❓ General Questions
Whether there are plans to support QWEN2-VL format？

**Comment by digisomni** [comment]
I'd like this as well. We have lots of vision models rolling out but no surefire way to run them with MLC.

**Comment by atlury** [comment]
Yes, a lot of us would like to have the vision models support.....

**Comment by yunho-c** [comment]
bump
